Expose `stylableSeparateCss` configuration (#2010)
wix/yoshi: docs/api/configuration.md

- ![status experimental](https://img.shields.io/badge/status-experimental-ff69b4.svg)

## separateStylableCss

> By default, the Stylable output will be bundled to the JS bundle (and rendered at runtime). You can bundle this output into a separate `app.stylable.bundle.css` file instead, by adding the following to your `package.json`:

Reviewer: ```suggestion
Output the stylable css into `app.stylable.bundle.css` file.

> By default, the Stylable CSS output will be bundled to the JS bundle (using the [`includeCSSInJS` option](https://github.com/wix/stylable/tree/master/packages/webpack-plugin#plugin-configuration-options)).
```
